library/NotificationCenter/MessageDraft/EmailMessageDraft.php

```diff
@@ -170,6 +170,14 @@ public function getHtmlBody()
     }
 
     /**
+     * Returns true if images should not be embedded
+     * @return boolean
+     */
+    public function useExternalImages() {
+        return !!$this->objLanguage->email_external_images;
```

Comment:
I wanted this to return a boolean, not "" or "1".
Not that you ask I'm not even sure anymore if Contao does this conversion from CHAR(1) to boolean on its own. :)